LR_Tourist/BLL/Model/Hotel.cs

```diff
@@ -1,19 +1,19 @@
 ﻿
 namespace BLL.Model
 {
-    public class Hotel: DA.Data.IEntityBase
+    public class Hotel: IEntityBase
     {
-        public int Id { get; }
+        public int Id { get; set; }
 
-        public string Name { get; set; }
+        public string NameHotel { get; set; }
```

Comment:
Why do you rename this field?
The previous version looks better.
We ca understand that the Name field store the hotel's name because the entity name is Hotel.
Revert this change.